src/sorting/usage.jl

```diff
@@ -0,0 +1,20 @@
+using BenchmarkTools
```

Comment:
This shouldn't be part of CUDA.jl. You can add it to the `perf/` benchmarks too, although I would there test the invocation of `Base.sort`, not `CUDA.quicksort` specifically.